jekyll/_cci2/non-aws.md

```diff
@@ -32,7 +32,7 @@ By default, CircleCI 2.0 Nomad Client instances automatically provision containe
 
 ## Architecture
 
-CircleCI Static consists of two primary components: Services and Nomad Clients. Services run on a single instance that is comprised of the core application, storage, and networking functionality. Any number of Nomad Clients execute your jobs and communicate back to the Services. Both components must access your instance of GitHub or GitHub Enterprise on the network as illustrated in the following architecture diagram.
+CircleCI Static consists of two primary components: Services and Nomad Clients. Services run on a single instance that is comprised of the core application, storage, and networking functionality. Any number of Nomad Clients (a.k.a Builder Fleet) execute your jobs and communicate back to the Services. Both components must access your instance of GitHub or GitHub Enterprise on the network as illustrated in the following architecture diagram.
```

Comment:
Builder in CircleCI refers to the 1.0 hosts that run the circlecircle application to create LXC or VMware execution environments. Nomad Clients should not be referred to as "builders" because it's added ambiguity.

Reply:
Makes sense, do we have an updated architecture image to use then and we can drop that language?

Reply:
I will request an update to this image from the illustrator, just change builders to Nomad Clients, correct?

Reply:
Ok, I have a request for the image update, when I get that later this week, I'll replace it.